tsg/ranker/base.py

```diff
@@ -96,9 +95,10 @@ def or_score_calc(query_terms, index_dictionary_path=DICTIONARY_PATH,
     query_length = {} # Holds score^2 for Length normalization at end
     doc_length = {}
     for term in query_terms:
-        query_term_weight = calculate_query_term_weight(term,query_terms,
-            index_dictionary_path, index_info_path)
         term_documents = get_dictionary_term_list(term, index_dictionary_path)
```

Comment:
I don-t understand why you see this code. If you check the file doc_freq is also there.